parlai/core/dict.py

```diff
@@ -292,6 +292,9 @@ def __init__(self, opt: Opt, shared=None):
                 # don't check isfile first, should fail if file not found
                 self.load(opt['dict_initpath'])
 
+        # cache unk token for later
```

Comment:
All the changes in this file caused small speed ups in tokenization. They're more pronounced in dynamic batching, since we may have to tokenize more examples between each parley, causing more overhead per GPU-hit.